src/components/InventoryGroups/Modals/AddSystemsToGroupModal.js

```diff
@@ -0,0 +1,208 @@
+import {
+    Alert,
+    Button,
+    Flex,
+    FlexItem,
+    Modal
+} from '@patternfly/react-core';
+import { fitContent, TableVariant } from '@patternfly/react-table';
+import difference from 'lodash/difference';
+import map from 'lodash/map';
+import PropTypes from 'prop-types';
+import React, { useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGroupDetail, selectEntity } from '../../../store/inventory-actions';
+import InventoryTable from '../../InventoryTable/InventoryTable';
+import { addHostsToGroupById } from '../utils/api';
+import apiWithToast from '../utils/apiWithToast';
+import ConfirmSystemsAddModal from './ConfirmSystemsAddModal';
+
+export const prepareColumns = (initialColumns) => {
+    // hides the "groups" column
+    const columns = initialColumns;
+
+    // additionally insert the "update methods" column
+    columns.splice(columns.length - 2 /* must be the 3rd col from the end */, 0, {
```

Comment:
Isn't it better to filter out the column instead of removing it with the splice method? Because, this can easily get affected if there is another new column added

Reply:
@mkholjuraev, hm, I am not removing any columns here from the default set. The only thing which this func is doing is inserting the "Update methods" column as the 3d from the end. This shouldn't be broken if new columns are added (although if the default columns number will be decreased to 3 or less then yeah there will be a problem 😄 but it's the least probable)

Reply:
well, the comment above the line 22 is misleading, it was copy-pasted from another code. I'll remove it